builtin/game/chatcommands.lua

```diff
@@ -15,6 +15,29 @@ function core.register_chatcommand(cmd, def)
 	core.registered_chatcommands[cmd] = def
 end
 
+function core.unregister_chatcommand(name)
+	if not core.registered_chatcommands[name] then
+		core.log("warning", "Not unregistering chatcommand " ..name..
+			" because it doesn't exist.")
+		return
+	end
+	core.registered_chatcommands[name] = nil
+end
+
+function core.override_chatcommand(name, redefinition)
+	if redefinition.name ~= nil then
+		error("Attempt to redefine name of "..name.." to "..dump(redefinition.name), 2)
+	end
+	local chatcommand = core.registered_chatcommands[name]
+	if not chatcommand then
+		error("Attempt to override non-existent chatcommand "..name, 2)
```

Comment:
I copied that sentence directly from `minetest.override_item` just changing `item` to `chatcommand`. IMO, if "Attempt to" is removed here, it should be there as well.